### PR TITLE
Added an example for styled paragraphs in PDF

### PR DESCRIPTION
**Pull request by danishyasin33**
Added the example for styled paragraphs in PDF. The example highlights how to use styled paragraphs to create different styles in a single paragraph. 
Addresses: https://github.com/unidoc/unipdf-examples/issues/78

### CODE REVIEW

File: report/styled_par.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
+/*
+ * This example showcases the creation of styled paragraph
+ * The output is saved as styled_paragraph.pdf which illustrates some of the features
+ * of the creator.
+ */
+/*
+ * NOTE: This example depends on github.com/unidoc/unipdf/v3/creator, MIT licensed,
+ *       and github.com/unidoc/unipdf/v3/model,
```
Comment (gunnsth) [inline report/styled_par.go]: This is incorrect.  unipdf is AGPL licensed with a commercial clause.  No need to include that here. Please keep consistent with other examples here.

File: report/styled_par.go [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import (
+	"log"
+
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func main() {
+
```
Comment (gunnsth) [inline report/styled_par.go]: Remove extra newline.  Keep style consistent with https://github.com/unidoc/unipdf/wiki/UniPDF-Developer-Guide
also run golint and goimports on the code.

File: report/styled_par.go [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+	c := creator.New()
+	c.NewPage()
+
+	fontRegular, _ := model.NewStandard14Font(model.HelveticaName)
+	fontBold, _ := model.NewStandard14Font(model.HelveticaBoldName)
+
+	//Styled paragraphs ***********************************************
```
Comment (gunnsth) [inline report/styled_par.go]: Clean up comment style.  We don't use *** like this normally.  Keep consistent.  And there should be a space between the // and S...

File: report/styled_par.go [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+	//Setting heading style
+	stChap := c.NewChapter("Styled Paragraphs")
+	stChap.GetHeading().SetMargins(0, 0, 20, 0)
+	stChap.GetHeading().SetFont(fontBold)
+	stChap.GetHeading().SetFontSize(18)
+	stChap.GetHeading().SetColor(creator.ColorRed)
+
+	//Creatign new styled paragraph
```
Comment (gunnsth) [inline report/styled_par.go]: Creating (typo)

File: report/styled_par.go [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+	stylPar := c.NewStyledParagraph()
+	stylPar.SetLineHeight(3) //setting line height
+
+	boldStyle := c.NewTextStyle() //creating new style
+
+	boldStyle.Font = fontBold                                                                                       //setting chunk's font style
```
Comment (gunnsth) [inline report/styled_par.go]: Can skip the reduntant comments on the right (pretty obvious)

File: report/styled_par.go [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+	boldStyle.Color = creator.ColorGreen                                                                            //setting chunk's color
+	chunk := stylPar.Append("This text is bolded and is in green color. We're showing how styled paragraphs work.") //setting chunk's text
+	chunk.Style = boldStyle                                                                                         //setting chunk's style
+
+	normStyle := c.NewTextStyle()                                                                                                                                               //creating new style
```
Comment (gunnsth) [inline report/styled_par.go]: We usually have comments above the code.  Please check the other examples, style guide and unipdf codebase and try to keep consistent.
No need to add comments that are obvious.

File: report/styled_par.go (hunk 1 above)
Comment (danishyasin33) [reply report/styled_par.go]: Done

File: report/styled_par.go (hunk 2 above)
Comment (danishyasin33) [reply report/styled_par.go]: Removed

**Pull request merged**